persist/src/main/resources/db/migration/V1__Initial_Import.sql

```diff
@@ -1,25 +1,60 @@
-CREATE TABLE IF NOT EXISTS member (
-  id VARCHAR(36) NOT NULL PRIMARY KEY,
-  email VARCHAR(36),
-  first_name VARCHAR(36),
-  last_name VARCHAR(36),
-  graduation_year INT,
-  major VARCHAR(36),
-  privilege_level INT
+CREATE TABLE IF NOT EXISTS users
+(
+    id              BIGSERIAL   NOT NULL PRIMARY KEY,
+    created_at      TIMESTAMP   NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at      TIMESTAMP   NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    deleted_at      TIMESTAMP,
+    email           VARCHAR(36) NOT NULL,
+    email_verified  BOOLEAN     NOT NULL DEFAULT FALSE,
+    first_name      VARCHAR(36) NOT NULL,
+    last_name       VARCHAR(36) NOT NULL,
+    privilege_level VARCHAR(16) NOT NULL,
```

Comment:
I think we may be able to enforce a custom constraint on this column which only allows two values

Reply:
Yes I believe this is done in the service `pom.xml`